mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -183,6 +183,26 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
       case Row(features: Vector, expected: Vector) =>
         assert(features ~== expected absTol 1e-14)
     }
+
+    // CountVectorizer test
+    val df2 = sqlContext.createDataFrame(Seq(
```

Comment:
Only add "d" does not work, because the index is not in the order of "a" "b" "c" "d". It is "a" "c" "b" "d" order. So I increase the counts of "b" to make it "a" "b" "c" "d". 

My change should not influence the order of vocabulary after fit.